### PR TITLE
mysql-stats rewrite

### PR DESCRIPTION
**Pull request by crcro**
- python3 
- new apps json data export

### CODE REVIEW

**Comment by murrant** [comment]
Looks good, is there a matching PR to handle this in LibreNMS?

**Comment by crcro** [comment]
I will have an update to this commit to handle diffs between reads and some innodb insert buffer changes (the output from SHOW INNODB ENGINE STATUS is a bit different now).

i will make the PR to handle this in LibreNMS soon (mostly done)

**Comment by SourceDoctor** [comment]
i miss a configfile to configure login credentials to database

File: snmp/mysql-stats [hunk 1]
```diff
@@ -57,75 +78,113 @@ cursor = conn.cursor()
...
+
+    new_entries = dict(zip(new_points_label, new_points_data))
+else:
+    new_entries = json.loads(str(entries).replace("\'","\""))
+
+with open(temp_file, 'w') as file:
+    file.write(str(entries))
+
+export = {"data": new_entries, "error": "0", "errorString": "", "version": "1"}
+data = re.sub(' ', '', json.dumps(export))
```
Comment (SourceDoctor) [inline snmp/mysql-stats:189]: space removement not needed
Example:
https://github.com/librenms/librenms-agent/pull/258/files

**Pull request closed without merging**